Assets/Creatures/api/PlayableCreatureBase.cs

```diff
@@ -85,5 +92,54 @@ public void Unsubscribe(DisabledState disabledState)
         public abstract AbilityExecutionResult CharacterAbility(CharacterAbilityEvent characterAbilityEvent, Vector3 mousePosition);
         public abstract AbilityExecutionResult MobilityAbility(MobilityAbilityEvent mobilityAbilityEvent, Vector3 mousePosition);
         public abstract AbilityExecutionResult CoreInfusionAbility(CoreInfusionAbilityEvent coreInfusionAbilityEvent, Vector3 mousePosition);
+
+        public virtual void HandleMovement(Vector2 inputDirection)
+        {
+            // Initialize movement based on input
+            Vector3 movement = Vector3.zero;
+
+            if (inputDirection.y > 0 && inputDirection.x > 0) // W + D
```

Comment:
I hate this sorta code but so necessary... could we assign these to vars like `inputUp, inputDown, inputLeft, inputRight` ?

Reply:
Hmm what are your thoughts on like:
```
public boolean isUp(float inputDirection) {
    return inputDirection.y > 0;
}
```

then we can have
```
if( isUp(inputDirection) && isRight(inputDirection)) // W + D
```

Reply:
https://github.com/FWFsoft/alien-arena/issues/21

Reply:
any of those work, but a boolean I think just looks cleaner if you assign all 4 of them.

isUp ...
isDown ...
isLeft ...
isRight ...

but functions work fine too